src/index.coffee

```diff
@@ -63,48 +67,56 @@ module.exports = class GeordiClient
   ###
   log event with Geordi v2.1
   ###
-  _logToGeordi: (eventData) =>
-    $.ajax {
-      url: @GEORDI_SERVER_URL,
-      type: 'POST',
-      contentType: 'application/json; charset=utf-8',
-      data: JSON.stringify(eventData),
-      dataType: 'json'
-    }
+  _logToGeordi: (eventData) ->
+    request = new XMLHttpRequest()
+    request.open "POST", @GEORDI_SERVER_URL[@env]
+    request.setRequestHeader "Content-Type", "application/json; charset=utf-8"
+    
+    new Promise (resolve, reject) ->
+
+      request.onload = () ->
+          if request.status == 200
+            resolve request.response
+          else
+            reject Error request.statusText
+
+      request.onerror = () ->
+          reject Error "Network Error"
```

Comment:
Let's make this a bit more descriptive:
"Network Error while attempting to log with Geordi"
or similar